### PR TITLE
Adding new definitions in the documentation

### PR DESCRIPTION
**Pull request by alexjoybc**
# Description

adding new definitions

### CODE REVIEW

File: docs/dictionary.md [hunk 1]
```diff
@@ -2,20 +2,28 @@
 
 ## Business
 
+[court definition](http://www.bclaws.ca/civix/document/id/complete/statreg/168_2009_01#subrule_d2e71)
+
 ### Document
 
+document has an extended meaning and includes a photograph, film, recording of sound, any record of a permanent or semi-permanent character and any information recorded or stored by means of any device;
```
Comment (sdevalapurkar-bcgov) [inline docs/dictionary.md:9]:
```suggestion
A document has an extended meaning and includes a photograph, film, recording of sound, any record of a permanent or semi-permanent character and any information recorded or stored by means of any device.
```

File: docs/dictionary.md [hunk 2]
```diff
@@ -2,20 +2,28 @@
...
+
 ### Efiling
 
-Electronic filing is the process of submitting documents over the Internet using jag efiling software that has been approved by `placholder`, such as a Restriction Order.
+Electronic filing is the process of submitting a package to a Court Registry over the Internet using jag efiling software that has been approved by Court Services Branch and the Judicary.
```
Comment (sdevalapurkar-bcgov) [inline docs/dictionary.md:13]:
```suggestion
Electronic filing is the process of submitting a package to a Court Registry over the Internet using the jag-efiling software that has been approved by the Court Services Branch and the Judicary.
```

File: docs/dictionary.md [hunk 3]
```diff
@@ -2,20 +2,28 @@
...
 
 ### File
 
 See [computer file](https://en.wikipedia.org/wiki/Computer%20file)
 
 ### Package
 
+A package can contain one or multiple documents that have or beeing submitted electronically for a single court file number.
```
Comment (sdevalapurkar-bcgov) [inline docs/dictionary.md:21]:
```suggestion
A package can contain one or multiple documents that have or are being submitted electronically for a single court file number.
```

File: docs/dictionary.md [hunk 4]
```diff
@@ -2,20 +2,28 @@
...
+
 ### Submission
 
+A submission is a set of documents, fees, comments that are beeing transmitted electronically for review and filing.
```
Comment (sdevalapurkar-bcgov) [inline docs/dictionary.md:25]:
```suggestion
A submission is a set of documents, fees, or comments that are being transmitted electronically for review and filing.
```

File: docs/dictionary.md [hunk 5]
```diff
@@ -2,20 +2,28 @@
...
+[court definition](http://www.bclaws.ca/civix/document/id/complete/statreg/168_2009_01#subrule_d2e71)
+
 ### Document
 
+A document has an extended meaning and includes a photograph, film, recording of sound, any record of a permanent or semi-permanent character and any information recorded or stored by means of any device.
+
 ### Efiling
 
-Electronic filing is the process of submitting documents over the Internet using jag efiling software that has been approved by `placholder`, such as a Restriction Order.
+Electronic filing is the process of submitting a package to a Court Registry over the Internet using the jag-efiling software that has been approved by the Court Services Branch and the Judicary.
```
Comment (akroon3r) [inline docs/dictionary.md:13]: jag-efiling application? not a blocker.

**Comment by akroon3r** [comment]
@sdevalapurkar-bcgov on you to approve

**Pull request merged**